modules/theme-tools/compat/twentytwenty.css

```diff
@@ -280,3 +280,17 @@
 .entry-content img.latex {
 	display: inline;
 }
+
+/**
+ * .com Toolbar
+ */
+
+/* fix wordpress.com notification iframe width */
+iframe#wpnt-notes-iframe2.wide {
+	width: auto !important;
```

Comment:
If you want to avoid use of `!important`, you can add the `intrinsic-ignore` class to the `#wpnt-notes-iframe2` iframe. When that class is present on an element or its parent, TwentyTwenty will skip trying to resize it.

Reply:
As this PR is merged and closed, I recommend opening a new PR with your proposed improvements.